app/account/connections/page.tsx

```diff
@@ -0,0 +1,54 @@
+'use client'
+
+import React from 'react';
+import sanitizeHtml from 'sanitize-html';
+import ConnectionSection from '@/components/ConnectionSection';
+import ContentContainer from '@/components/ContentContainer';
+import PageWrapper from '@/components/PageWrapper';
+
+const REDIRECT_URI = process.env.NEXT_PUBLIC_ORCID_DEV_CALLBACK;
+const ORCID_CLIENT_ID = process.env.NEXT_PUBLIC_ORCID_CLIENT_ID;
+
+const ConnectionsPage: React.FC = () => {
+
+  // Sandbox Uri
+  //const orcidUri = `https://sandbox.orcid.org/oauth/authorize?client_id=${orcidClientId}&response_type=code&scope=/read-limited&redirect_uri=${redirectURI}`;
```

Comment:
Do we want to leave this? Maybe better to have an env check and use the appropriate one?
We can use the sandbox for dev and the other for stage/prod

Reply:
@briri I will be removing that later actually, because the authorization will be happening on the backend. There's a ticket for that: https://github.com/CDLUC3/dmsp_backend_prototype/issues/116